src/async-wrap.cc

```diff
@@ -791,3 +792,56 @@ void EmitAsyncDestroy(Isolate* isolate, async_context asyncContext) {
 }  // namespace node
 
 NODE_MODULE_CONTEXT_AWARE_BUILTIN(async_wrap, node::AsyncWrap::Initialize)
+
+
+// Only legacy public API below this line.
+
+namespace node {
+
+MaybeLocal<Value> MakeCallback(Isolate* isolate,
```

Comment:
No particular reason, except maybe that it kind of belongs to the rest of the legacy code here.